chapter07/sxg/ex_7_6.pl

```diff
@@ -0,0 +1,27 @@
+#!perl
+use warnings;
+use strict;
+
+my $findW=0;
+my $findF=0;
+my $line;
+
+print "I found:\n";
+while (<>) {
+	$line = $_;
+
+  	if (/wilma/) {
+  		$findW = 1;
+	}
+  	if (/fred/) {
+  		$findF = 1;
+	}
+	if($findF && $findW){
+		print "$line";	
+	}

```

Comment:
This seems like a interesting way to do it but do $findF and $findW get returned to zero each loop? If not once fred and wilma have both appeared(even on different lines) would every line not be printed?